src/entry-points/local-file-player/App.svelte

```diff
@@ -45,14 +44,14 @@ along with Jump Cutter Browser Extension.  If not, see <https://www.gnu.org/lice
     }
 
     for (const file of inputEl.files) {
-      files.push(file);
+      files = [...files, file];
     }
 
     playFile(0);
   }
   async function playFile(ind: number) {
     currFileInd = ind;
-    const file = inputEl.files[ind];
+    const file = files[currFileInd];
```

Comment:
assign target play file with current file index

Reply:
Not sure why you changed it from `ind` to `currFileInd`.

Reply:
I mean if you do want to change it, then it should be in a separate "refactor" commit.